component/webui/src/main/java/org/exoplatform/social/webui/space/UISpaceSettings.java

```diff
@@ -48,7 +48,7 @@ public UISpaceSettings(String name) throws Exception {
     //
     addUIFormInput(new UIFormStringInput(SPACE_DISPLAY_NAME, SPACE_DISPLAY_NAME, "").
                    addValidator(MandatoryValidator.class).
-                   addValidator(ExpressionValidator.class, "^([\\p{L}\\s\\d\']+[\\s]?)+$", MSG_INVALID_SPACE_NAME).
+                   addValidator(ExpressionValidator.class, "^([\\p{L}\\d\']+[\\s]?)+$", MSG_INVALID_SPACE_NAME).

```

Comment:
This change aligns the regex with the one in UISpaceInfo, so it is a good thing, but I am not sure to understand it completely. It seems more complex than necessary. We allow only letters, digits, spaces and single quotes, so why not simply "^[\p{L}\d\s']+$" ?